app/controllers/clients.js

```diff
@@ -5,6 +5,18 @@ const Conversations = require('../models/conversations');
 const Messages = require('../models/messages');
 const Users = require('../models/users');
 
+function average (arr) {
```

Comment:
Well, the pattern I have kind of arbitrarily chosen is to put extra methods that aren't used outside the file as underscored methods on the module.exports object. So it would be _average(), but on the actual exported object. 

Since this is a function that takes an average of an array it likely would have a better home in a lib file somewhere.